docs/user/managing-fleets.md

```diff
@@ -0,0 +1,117 @@
+2. A **device template** that is the template for the device specifications of all devices in the fleet (see [Defining Device Templates](managing-fleets.md#defining-device-templates)).
+3. A set of **policies** that govern how devices are managed, for example how changes to the device template are rolled out to devices (see [Defining Rollout Policies](managing-fleets.md#defining-rollout-policies)).
+
+When a user updates a fleet's device template to a new version, a component called "fleet controller" starts the process of rolling out this version to all of the fleet's devices over time. When the controller has selected a device for update, it copies the new device template to the device's specification. The next time the device's agent checks in, it learns of the new specification and begins the update process.
+
+The same thing happens when an individually managed device joins a fleet or a device changes fleets: The device template is copied to the device's specification and if that specification differs from the previous one as a result, the agent will perform an "update".
+
+## Selecting Devices into a Fleet
+
+In Flight Control, devices are not explicitly assigned to a fleet. Instead, each fleet has a "selector" that defines which labels a device must have to be selected into the fleet. This allows decoupling the concerns of organizing devices from operating them.
+
+Let us have a look at a practical example. Assume the following list of point-of-sales (PoS) terminal devices and their labels:
+
+| Device | Labels |
+| ------ | ------ |
+| A | type: pos-terminal, region: east, stage: production |
+| B | type: pos-terminal, region: east, stage: development |
+| C | type: pos-terminal, region: west, stage: production |
+| D | type: pos-terminal, region: west, stage: development |
+
+If all PoS terminals used more or less the same configuration and were managed by the same operations team, it could make sense to define a single fleet `pos-terminals` with a label selector `type=pos-terminal`. The result is that the fleet would contain all devices A, B, C, and D.
+
+Often, though, you would have separate organizations developing solutions and deploying and operating them. In this case, it could make sense to define two fleets `development-pos-terminals` with label selector `type=pos-terminal, stage=development` that selects devices C and D and similar for the production PoS terminals. This way, fleets can be managed independently.
+
+Note that you have to define selectors so that no two fleets select the same device. Say you had one fleet select `region=east` and another `stage=production`, then both would select device A. Flight Control would detect this conflict and would set the "OverlappingSelectors" condition on both fleets to "true", which suspends any automation of these fleets until the problem is fixed.
```

Comment:
> which suspends any automation of these fleets until the problem is fixed.

This isn't how it works today.  It's more of a warning.  Device A will be selected by one of the fleets first.  Then, when the other fleet tries to take ownership, we'll set the Conditions.  However, device A will remain in its original fleet and operations will be allowed to continue.  We can discuss changing this behavior, but that's what it is right now.

Reply:
Thanks. I don't want to change behaviour, just misunderstood how it works. Fixed.